packages.x86_64

```diff
@@ -122,15 +119,13 @@ xfsprogs
 xl2tpd
 zsh
 
-# virbos
-#maxfetch
 # bspwm
 sway
 dmenu
 alacritty
 firefox
 xorg-xwayland
-xorg-xinit
```

Comment:
Why are you removing xinit?

Reply:
Since the ISO now runs sway with wayland it is no longer required